users/userModel.js

```diff
@@ -0,0 +1,30 @@
+const mongoose = require('mongoose');
+const bcrypt = require('bcrypt');
+const ObjectId = mongoose.Schema.Types.ObjectId;
```

Comment:
I don't think you'll need this declaration for this project (I don't think we'll be modeling relationships between documents here), but good work remembering it. Better to have it and not need it than to need it and not have it! 😁